libcloud/__init__.py

```diff
@@ -24,6 +24,8 @@
 import codecs
 import atexit
 
+import requests
```

Comment:
We import this file in ``setup.py`` to read ``__version__`` attribute from it which means ``setup.py`` calls could fail if ``requests`` is not installed on the system (aka when setup.py is called during installations).

I think it would be safer and better for us to simply parse this file as string and use regex to match ``__version__`` attribute.

Something along those lines - https://github.com/StackStorm/st2/blob/master/scripts/dist_utils.py#L134

Reply:
In order to simplify the check and to stick with the existing pattern used for the paramiko library, I've changed the code to use a try/catch import. During `setup.py`, we don't particularly care about executing the requests version check (this can be delayed until first run of the library) so the functionality should be equivalent to what was there before. Let me know if this is okay.